ex2_header_css.css

```diff
@@ -0,0 +1,40 @@
+.header {
+	width: 100%;
+	height: 60px;
+	border-bottom: 1px solid;
+}
+
+.logintitle {
+	background-color: #3090C7;
+	padding: 3px;
+	text-align: center;
+	color: white;
+	box-shadow: 0px 5px 10px grey;
```

Comment:
Could this `0px` be written as `0%`? What about `0vh`? `0rem`? Would it make any difference?